modules/candidate_list/php/NDB_Menu_Filter_candidate_list.class.inc

```diff
@@ -85,29 +86,15 @@ class NDB_Menu_Filter_Candidate_List extends NDB_Menu_Filter
             array(
              'GROUP_CONCAT(DISTINCT sp.title) as Subproject',
              'DATE_FORMAT(c.DoB,\'%Y-%m-%d\') AS DoB',
-             'MAX(s.Scan_done) as scan_Done',
+             "CASE MAX(s.Scan_done) 
```

Comment:
Can we transfer this logic to columnFormater.js? There is already an if statement to add the link to imaging_browser.

Reply:
we could.. it makes more sense to me to be "Yes/No" in the JSON returned, but I don't feel strongly about it if you think it should be in the formatter.

Reply:
Truth comes from the database; even if it wrong.